packages/rocm-k8s-device-plugin/rocm-k8s-device-plugin.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Start ROCm kubernetes device plugin
+After=kubelet.service
+Wants=kubelet.service
+
+[Service]
+# Ensure that the kubelet device plugin socket exists before we start
+# A brief sleep is needed to avoid the `test` failing its first check
+ExecStartPre=/usr/bin/sleep 0.1
+ExecStartPre=/usr/bin/test -S /var/lib/kubelet/device-plugins/kubelet.sock
+ExecStart=/usr/bin/rocm-device-plugin -logtostderr=true -stderrthreshold=INFO -v=5
+Type=simple
+TimeoutSec=0
+RestartSec=2
+Restart=always
+StandardError=journal+console
```

Comment:
We shouldn't send most errors to the console, it's only for services that can cause boot to fail:
```suggestion
```